docs/cli/kops_replace.md

```diff
@@ -5,7 +5,7 @@ Replace cluster resources.
 ### Synopsis
 
 
-Replace a resource specification by filename or stdin.
+Replace a resource configuration by filename or stdin.
```

Comment:
Used "Replace a resource by filename" as proposed as replaced specification by "desired configuration" in the example.